packages/communication-react/scripts/verifyTypeTrimmedDeclaration.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Verify the 'types' field is correct in the package.json based on the supplied release type parameter.
+ *
+ * @command verify - this is the default. This checks the field is correct.
+ * @command update - this can be used to update the field to the correct string.
+ *
+ * @param releaseCandidate - this must be 'public', 'beta' or 'alpha'.
+ *
+ * @examples
+ *   > node .\verifyTypeTrimmedDeclaration.js verify --rc public
+ *   > node .\verifyTypeTrimmedDeclaration.js verify --rc beta
+ *   > node .\verifyTypeTrimmedDeclaration.js update --rc public
+ *
+ * @remarks
+ * Passing the following parameters should result in the following checks:
+ *   * 'public' - "types": "dist/<package.name>-public.d.ts"
+ *   * 'beta' - "types": "dist/<package.name>-beta.d.ts"
+ *   * 'alpha' - "types": "dist/<package.name>-untrimmed.d.ts"
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { exit } = require('process');
+const yargs = require('yargs');
+
+// Hard code constant values. These would need dynamically fetched from the package.json if
```

Comment:
nitty nit: Would drop this comment. I don't expect this script to ever be used outside of this package (and there will be other problems besides). :)

This is a very concrete package-contextual tool.

Reply:
updated